src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -332,58 +161,6 @@ export const CONTROL_EVENTS = {
     CAMERA_TARGET_CHANGED: 'camera-target-changed',
 };
 
-
-// SnapCamera saves transformation's camera
-// It's use to globe move
-function SnapCamera(camera) {
-    camera.updateMatrixWorld();
-
-    this.matrixWorld = new THREE.Matrix4();
-    this.projectionMatrix = new THREE.Matrix4();
-    this.invProjectionMatrix = new THREE.Matrix4();
-    this.position = new THREE.Vector3();
-
-    this.init = function init(camera) {
-        this.matrixWorld.copy(camera.matrixWorld);
-        this.projectionMatrix.copy(camera.projectionMatrix);
-        this.position.copy(camera.position);
-        this.invProjectionMatrix.getInverse(this.projectionMatrix);
-    };
-
-    this.init(camera);
-
-    this.shot = function shot(objectToSnap) {
-        objectToSnap.updateMatrixWorld();
-        this.matrixWorld.copy(objectToSnap.matrixWorld);
-        this.position.copy(objectToSnap.position);
-    };
-
-    const matrix = new THREE.Matrix4();
-
-    this.updateRay = function updateRay(ray, mouse) {
-        ray.origin.copy(this.position);
-        ray.direction.set(mouse.x, mouse.y, 0.5);
-        matrix.multiplyMatrices(this.matrixWorld, this.invProjectionMatrix);
-        ray.direction.applyMatrix4(matrix);
-        ray.direction.sub(ray.origin).normalize();
-    };
-}
-
-var snapShotCamera = null;
-
-function defer() {
-    const deferedPromise = {};
-    deferedPromise.promise = new Promise((resolve, reject) => {
-        deferedPromise.resolve = resolve;
-        deferedPromise.reject = reject;
-    });
-    return deferedPromise;
-}
-
-let initPromise;
-
-/* globals document,window */
-
 /**
  * @class
  * @param {GlobeView} view
```

Comment:
Please add a description of the options, in the same idea as in `PlanarControls`.